packages/compass-components/src/components/workspace-tabs/tab.tsx

```diff
@@ -158,11 +170,15 @@ const tabSubtitleStyles = css({
 });
 
 const closeButtonStyles = css({
-  visibility: 'hidden',
+  transition: tabTransition,
+  transitionProperty: 'opacity, transform',
+  transform: 'translateY(44px)',
```

Comment:
Not sure about this whole animation yet. Trying to find the code used in the figma design.